src/test/java/org/apache/ibatis/submitted/parametrizedlist/User.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.apache.ibatis.submitted.parametrizedlist;
 
-public class User<T> {
+public class User {
```

Comment:
It may be intentional to use parameterized. Hi @harawata , Is this change appropriate?

Reply:
@kazuki43zoo is right. This is intentional and we need to keep the type parameter.

Reply:
@yangfancoming please revert this changes.

Reply:
sorry， I had revert  that commit    please take a look.

Reply:
Sorry, I am a novice, I have revert that commit, but I do not know if the operation is correct. please take a look.